reconcile/terraform_vpc_peerings.py

```diff
@@ -431,8 +429,8 @@ def run(dry_run, print_only=False,
     ts.populate_vpc_peerings(desired_state)
     working_dirs = ts.dump(print_only=print_only)
 
-    if print_only:
-        sys.exit()
+    if print_only or any(errors):
+        sys.exit(0 if dry_run else 1)
```

Comment:
i think this introduces unwanted behavior. we only want to exit if this is a print-only.
i would change this line to `sys.exit(int(any(errors)))` like you did below, but not change the condition.